browser/src/components/pages/Information.vue

```diff
@@ -224,7 +226,7 @@
         </div>
         <div
           class="flex justify-center my-md"
-          style="background:#6096BA"
+          style="background:#6096BA; border-radius:35px"
```

Comment:
better keep to tailwind syntax https://tailwindcss.com/docs/border-radius

custom values can be passed with `border-[35px]` for example